app/src/app/components/view-profile/view-profile.component.ts

```diff
@@ -166,4 +167,12 @@ export class ViewProfileComponent implements OnInit {
       this.populateForm(this.personDetails);
     }
   }
+
+  isSaveButtonDisabled(): boolean {
+    if (!this.profileForm.dirty) {
```

Comment:
should be just a check for "valid", not "dirty"

Reply:
done